src/lib/docs.ts

```diff
@@ -90,17 +92,28 @@ const serializeSignature = ({
   checker: TypeChecker;
   signature: Signature;
 }): DocEntryConstructor => {
-  return {
+  const result: DocEntryConstructor = {
```

Comment:
What do you think about omitting `visibility` here?

```
const result: Omit<DocEntryConstructor, "visibility"> = {
   ...
   
   if (!signature.declaration) {
    return {
       ...result;
       visibility: 'public'
    };
  }
  ```

Reply:
this is possible, but if it's the default value on the return, i think it makes sense to include it in the declaration of `result`. I do think your suggestion is more human readable / easier to maintain, so will change.